### ISSUE TITLE
Container closing unexpectedly

### ISSUE THREAD

**Issue by BoatNectar** [cemizm/tf-benchmark-gpu#2]
Running the following command, the container exits upon reaching the running warmup step. I've omitted the runtime=nvidia flag as it appears to no longer be supported. If there's any more information I can share to help pin down the issue I'm having I'd be happy to share.

$ sudo docker run  --rm cemizm/tf-benchmark-gpu --model resnet50 --num_gpus=1
TensorFlow:  1.14
Model:       resnet50
Dataset:     imagenet (synthetic)
Mode:        training
SingleSess:  False
Batch size:  64 global
             64 per device
Num batches: 100
Num epochs:  0.00
Devices:     ['/gpu:0']
NUMA bind:   False
Data format: NCHW
Optimizer:   sgd
Variables:   parameter_server
==========
Generating training model
Initializing graph
Running warm up
$
